Class Presentations/reveal-presentations/SelfConf2016.html

```diff
@@ -0,0 +1,582 @@
+				 need empathetic connections, not judgement.
+			</aside>
+			</section>
+
+			<section>
+				<h2>Goals</h2>
+				<ul class="checklist">
+					<li>
+						<input type="checkbox" id="goal0" checked />
+						<label for="goal0">Who Are We, and What Is This Project?</label>
+					</li>
+					<li>
+						<input type="checkbox" id="goal1" checked />
+						<label for="goal1">Why Did We Decide to Do This?</label>
+					</li>
+					<li>
+						<input type="checkbox" id="goal2" checked />
+						<label for="goal2">What Resources Did We Use?</label>
+					</li>
+					<li>
+						<input type="checkbox" id="goal3" checked />
+						<label for="goal3">What Has Been Easy? Difficult? Fun?</label>
+					</li>
+					<li>
+						<input type="checkbox" id="goal4" checked />
+						<label for="goal4">What Are Some Things We Didn't Think About?</label>
+					</li>
+					<li>
+						<input type="checkbox" id="goal5" />
+						<label for="goal5">What Lessons Did We Learn?</label>
+					</li>
+				</ul>
+			</section>
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+			<section>
+				<h2>What Lessons Did We Learn?</h2>
+			</section>
+
+			<section>
+				<h2>Re: Organization</h2>
+				<ul>
+					<li>Get the academic calendar with all academic/extracurricular days</li>
+					<li>A stronger system for keeping track of your own files</li>
+					<li>Creating more integrated lessons that incorporated more soft skills
+						 from the beginning.</li>
+				</ul>
+				<aside class="notes">Academic calendar--so we could have planned for
+					testing/mid-terms/finals and ceremonies/celebrations as well as
+					vacations/days out.
+					File system--students switched computers a lot, or would forget flash drives
+					Integrated lessons--coding skills are not enough, they need to know what
+					being a coder is really like and what abilities are valuable in the industry
+				</aside>
+			</section>
+
+			<section>
+				<h2>Re: Engagement</h2>
+				<ul>
+					<li>Create more out of class lessons to be delivered via email/text
+						 messaging</li>
+					<li>Use of more hands-on student-led activities</li>
+				</ul>
+				<aside class="notes">Out of class lessons -- keep engagement high between sessions.
+					Hands-on activities--We'd want them to have to research
+					to complete the tasks vs following our examples. The students should be
+					driving their learning as often as possible
+				</aside>
+			</section>
+
+			<section>
+				<h2>Re: Efficiency</h2>
+				<ul>
+					<li>Have students take an entry survey</li>
+					<li>Develop on codepen.io (or something similar) from the beginning</li>
+				</ul>
+				<aside class="notes">Survey: to create a baseline we could properly
+					measure their development (in skill and interest) from
+					Codepen: so you can see how everything works together without the
+					saving and loading, etc. Eventually we'd teach them that, but we could
+					save the complication at the beginning
+				</aside>
+			</section>
+
```

Comment:
Should we have a slide for QA time? Also, how long is our talk supposed to be anyway?